third-party-bom/pom.xml

```diff
@@ -644,6 +644,16 @@
                 <mockito.version>5.5.0</mockito.version>
             </properties>
         </profile>
+        <profile>
+            <!-- prevents "bootstrap class path not set in conjunction with -source 8" -->
+            <id>set-compiler-release</id>
```

Comment:
please see the profile above created by @peter-lawrey 

@peter-lawrey I don't understand the purpose of the above profile so need you to review this change

Reply:
I suspect the line you are overriding should just be removed instead. Testing this locally.

Reply:
If we include JDK11 in this new profile then the existing above should be changed to `<jdk>(11,</jdk>` at least ?

Reply:
@peter-lawrey I tested this change with one of the demos and it worked like a charm. I am keen to get this merged

Reply:
This was solved in x.26 by setting -Xlint:-options

Reply:
Why have `-options` when instead we can do this and fix it properly, and we dont have to wait for x26?